styles.css

```diff
@@ -97,3 +97,378 @@ body {
+  text-align: center;
+}
+
+.icons > ul > li > a {
+  margin-right: 20px;
+  margin-top: 5px;
+  color: #505f79;
+  height: 20px;
+  width: 20px;
+}
+
+.my-work a:hover {
+  color: white;
+  background-color: #396df2;
+}
+
+.get-resume-btn:hover {
+  color: white;
+  background-color: #396df2;
+}
+
+.icons > ul > li > a:hover {
+  color: #7f8cff;
+}
+
+.accordation {
+  padding-right: 10px;
+}
+
+.section2 {
+  display: grid;
+  grid-template-columns: auto;
+  padding: 5%;
+}
+
+.my-work {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+  border-radius: 15px;
+  background-color: white;
+  padding: 5%;
+  margin-top: 80px;
+}
+
+.my-work img {
+  margin-bottom: 40px;
+}
+
+.my-work h2 {
+  margin: 0;
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: bold;
+}
+
+.my-work p {
+  margin: 0;
+  color: #344563;
+}
+
+.position-container {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.position-container span {
+  color: #344563;
+  font-weight: bold;
+  text-transform: uppercase;
+  font-size: 14px;
+}
+
+.position-container ul {
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+}
+
+.position-container li {
+  margin-right: 12px;
+  font-size: 13px;
+  color: #7a869a;
+  font-weight: bold;
+}
+
+.tech {
+  display: flex;
+  justify-content: flex-start;
+  margin: 8px 0;
+}
+
+.tech li {
+  display: inline-block;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  margin-right: 10px;
+  font-size: 12px;
+}
+
+.mother-accordion li {
+  list-style-type: none;
+}
+
+.child-accordion li {
+  background-color: #f7f7f9;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 15px;
+  padding: 10px;
+  border-radius: 8px;
+}
+
+.accordion {
+  margin: 60px 0;
+}
+
+.accordion-item {
+  display: block;
+  margin-bottom: 50px;
+  border-bottom: 1px solid #505f79;
+}
+
+.mother-accordion {
+  padding: 0;
+  margin: 0;
+}
+
+.mother-accordion h2 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.child-accordion {
+  margin: 0;
+  padding: 0;
+}
+
+.child-accordion div {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.child-accordion img {
+  margin-right: 20px;
+}
+
+.child-accordion span {
+  color: #253858;
+  font-weight: bold;
+}
+
+.social {
+  color: #7f8cff;
+  margin: 8px 0;
+}
+
+.social-links h2 {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.social-links li:hover {
+  color: #6070ff;
+}
+
+.get-resume > button {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
+  border-color: #7f8cff;
+  color: #7f8cff;
+  margin-top: 22px;
+  background-color: white;
+  border-radius: 8px;
+  display: flex;
+}
+
+.section3 {
+  background-color: white;
+  border-top-right-radius: 7rem;
+  margin-top: 20px;
+  padding: 20px 5%;
+}
```

Comment:
- [ ] This section is great and very similar to the Figma template, but it is a project requirement to style it with Flexbox. Please, add flexbox to this class to follow all the instructions.